syft/pkg/cataloger/alpm/parse_alpm_db.go

```diff
@@ -31,8 +31,13 @@ var (
 	}
 )
 
+type parsedData struct {
+	Licenses         string `mapstructure:"licenses"`
+	pkg.AlpmMetadata `mapstructure:",squash"`
+}
+
 func parseAlpmDB(resolver source.FileResolver, env *generic.Environment, reader source.LocationReadCloser) ([]pkg.Package, []artifact.Relationship, error) {
-	metadata, err := parseAlpmDBEntry(reader)
+	parsedData, err := parseAlpmDBEntry(reader)
```

Comment:
`parsedData` as a variable is shadowing a type name, which is not recommended (can be confusing to read)